pkgs/development/libraries/SDL_mixer/default.nix

```diff
@@ -1,4 +1,9 @@
-{ stdenv, lib, fetchurl, SDL, libogg, libvorbis, smpeg, enableNativeMidi ? false, fluidsynth ? null }:
+{ stdenv, lib, fetchurl
+, SDL, libogg, libvorbis, smpeg, libmikmod
+, fluidsynth
+, enableNativeMidi ? false
+, enableMusicMod ? false
```

Comment:
Is there any downside if we enabled this by default?

Reply:
Regarding enableNativiMidi: I don't know.

Regarding enableMusicMod (the flag introduced by me): Don't think so, apart from a slight increase in closure size (466 KiB).

Reply:
Can we remove enableMusicMod? If there is no downside except 466 KiB it is not worth it to keep it.

Reply:
Okay, will do so in a second! :-)